requirements.txt

```
@@ -0,0 +1,29 @@
+# NVDA's build system is SCons
+SCons==4.1.0.post1
+
+# NVDA's runtime dependencies
+comtypes==1.1.7
+pyserial==3.5
+wxPython==4.1.1

```

Please consider adding configobj from git:

```suggestion
wxPython==4.1.1
git+git://github.com/DiffSK/configobj@3e2f4cc#egg=configobj
```

Note, this is the last commit from current master, therefore this would require a readme update. No major changes